src/clj/kixi/hecuba/data/measurements.clj

```diff
@@ -225,6 +225,7 @@
   If fetch size is not provided, uses the default 100.
   Returns a lazy sequence of measurements."
   [store args & [opts]]
+  (assert (map? args) "Where clause passed must be a map!")
```

Comment:
I wonder if we should start using pre and post more. wdyt? Happy with the assert/core.typed/prismatic schema.